cli_tools/common/utils/daisyutils/daisy_utils.go

```diff
@@ -438,6 +438,15 @@ Loop:
 	return w, nil
 }
 
+// Tool is used to communicate the tool's name ot the user.
+type Tool struct {
+
+	// HumanReadableName is used for error messages, for example: "image import".
+	HumanReadableName string
+	// URISafeName is used programmatically, eg: "image-import"
+	URISafeName string
```

Comment:
How is this name being URI safe and being used programmatically related?

Reply:
Updated to `ResourceLabelName`

The original intention was to have one for programmatic use, and one for putting in logs, with URL safe being an indicator of which characters to use. That said, it's probably clearest to document exactly where it will be used, as with this new name.